project_event/__manifest__.py

```diff
@@ -38,6 +38,8 @@
         'views/activity_template_view.xml',
         'views/task_template_view.xml',
         'views/res_partner_category_type_view.xml',
+        'views/project_task_view.xml',
```

Comment:
You should put this model "res.partner.sector" and its view in project_resource_calendar

Reply:
@eilst : Current implementation does not have any issues at the comment. It is working as expected (spec requirement).